docker/Dockerfile

```diff
@@ -10,38 +29,20 @@ ARG PROJECT_VERSION=2.x
 # Set Apache ServerName directive globally to suppress AH00558 message.
 RUN echo "ServerName localhost" >> /etc/apache2/apache2.conf
 
+# Install and enable geos
+COPY --from=geos /usr/local/lib/php/extensions/ /usr/local/lib/php/extensions/
+RUN docker-php-ext-enable geos
+
 # Install the BCMath PHP extension.
 RUN docker-php-ext-install bcmath
 
-# Install git and unzip (needed to build PHP GEOS and needed by Composer).
-RUN apt-get update \
-  && apt-get install -y git unzip
-
-# Build and install the GEOS PHP extension.
-# See https://git.osgeo.org/gitea/geos/php-geos
-RUN apt-get update && apt-get install -y libgeos-dev \
-  && git clone https://github.com/libgeos/php-geos.git \
-  && ( \
-    cd php-geos \
-    # Checkout latest commit with PHP 8 support.
-    && git checkout e77d5a16abbf89a59d947d1fe49381a944762c9d \
-    && ./autogen.sh \
-    && ./configure \
-    && make \
-    && make install \
-  ) \
-  && rm -r php-geos \
-  && docker-php-ext-enable geos
-
 #Add custom configurations
 COPY conf.d/ /usr/local/etc/php/conf.d
 
-# Install postgresql-client so Drush can connect to the database.
 RUN apt-get update \
-  # See https://stackoverflow.com/questions/51033689/how-to-fix-error-on-postgres-install-ubuntu
-  && mkdir -p /usr/share/man/man1 \
-  && mkdir -p /usr/share/man/man7 \
-  && apt-get install -y postgresql-client
+    && apt-get install -y git unzip postgresql-client libgeos-c1v5 \
```

Comment:
Oh I just noticed that `git` and `unzip` installs moved down here. We need to preserve the reasoning for each package with comments. Perhaps separating them with newlines and adding a comment above each one? For example:

```
# Install apt dependencies.
RUN apt-get update && apt-get install -y \
        # Install git and unzip (needed by Composer).
        git unzip \
        # Install postgresql-client so Drush can connect to the database.
        postgresql-client \
        # Install libgeos-c1v5 ... ?
        libgeos-c1v5 \
    && rm -rf /var/lib/apt/lists/* \
    && apt-get clean
```

What is `libgeos-c1v5` for? We didn't have that before. Is that what you are referring to in the commit message with "Add missing dependency"? We should include more detail for future reference.